JS/fluentRevealNavbar.uc.js

```diff
@@ -10,6 +10,9 @@
 // ==/UserScript==
 
 (function() {
+  // store cached navbar buttons
+  let cachedButtons;
```

Comment:
I'm confused by this change. All you need to do to permanently cache toolbarButtons is this:

```js
    // get all the toolbar buttons in the navbar, in iterable form
    get toolbarButtons() {
      if (!this._toolbarButtons || !this._options.cacheButtons) {
        this._toolbarButtons = Array.from(
          gNavToolbox.querySelectorAll(".toolbarbutton-1")
        );
        if (this._options.includeUrlBar) {
          this._toolbarButtons.push(
            gNavToolbox.querySelector("#urlbar-background")
          );
        }
        if (this._options.includeBookmarks) {
          this._toolbarButtons = this._toolbarButtons.concat(
            Array.from(
              this.placesToolbarItems.querySelectorAll(".bookmark-item")
            )
          );
        }
      }
      return this._toolbarButtons;
    }
```

Also, `this._options.cacheButtons == false` bothers eslint. The code hasn't been formatted either. You should run `npm install` inside the repo so you can lint & format your changes.

Reply:
Yeah, sorry. I initially created the PR by manually copy-pasting parts of code to github.dev window to create the commit history.

Reply:
That's ok. If it's not easy for you to do linting and formatting, I can do it on a follow-up commit.

Reply:
It would be great! Guess I now push few more commits with your suggested changes